cypress/unit/store-mutations-setDataDictionary.cy.js

```diff
@@ -11,17 +14,37 @@ describe("setDataDictionary", () => {
 
             dataDictionary: {
 
-                "age": {
+                provided: {
```

Comment:
I don't think anything depends on this part of the datadictionary being mocked? Are you including it only as a way to remind users that this attribute also exists? Or is there something I'm missing?

I think if the goal is to have a pre-defined "new user generated data dictionary" be defined once, then I find it much clearer to have that be done directly in a separate variable like `newDataDictionary`.

Reply:
@surchs What are you referring to by "this part" here?

Reply:
@jarmoza i was referring to the 'userProvided' part you had mocked here